ext/native-decls/GetHeliBodyHealth.md

```diff
@@ -0,0 +1,18 @@
+---
+ns: CFX
+apiset: server
+game: gta5
+---
+## GET_HELI_BODY_HEALTH
+
+```c
+int GET_HELI_BODY_HEALTH(Vehicle heli);
+```
+
+**Note**: If any of the following natives: [SET_VEHICLE_BODY_HEALTH](https://docs.fivem.net/natives/?_0xB77D05AC8C78AADB), [SET_VEHICLE_ENGINE_HEALTH](https://docs.fivem.net/natives/?_0x45F6D8EEF34ABEF1), or [SET_VEHICLE_PETROL_TANK_HEALTH](https://docs.fivem.net/natives/?_0x70DB57649FA8D0D8) is used with a value greater than `1000`, the health data for the helicopter becomes accessible through all related getter natives, even if their specific values are set below `1000`. Setting any one of these natives above `1000` allows the others to be used with any value.

```

Comment:
```suggestion
**Note** This native will always return `1000.0` unless [SET_VEHICLE_BODY_HEALTH](#_0xB77D05AC8C78AADB), [SET_VEHICLE_ENGINE_HEALTH](#_0x45F6D8EEF34ABEF1), or [SET_VEHICLE_PETROL_TANK_HEALTH](#_0x70DB57649FA8D0D8) have been called with a value greater than `1000.0`.
```
This should simplify the wording and hopefully make it easier to understand. This should probably also be put on the natives repo as a disclaimer for these natives too.